deploy/ray-operator/controllers/common/constant.go

```diff
@@ -0,0 +1,27 @@
+package common
+
+const (
+	// use as pod type decision
+	Head = "head"
+
+	// use as label key
+	rayclusterComponent = "raycluster.component"
+	rayIoComponent      = "rayclusters.ray.io/component-name"
+	rayOperator         = "ray-operator"
+	RayClusterOwnerKey  = "raycluster.instance.name"
+	ClusterPodType      = "raycluster.pod.type"
```

Comment:
> For now these label keys used by the operator internally, it will be public if necessary.

Are these keys used in other places?

Reply:
Yes , these keys are used in the raycluster_controller.go